lib/client-api/src/client_api.js

```diff
@@ -203,7 +203,12 @@ export default class ClientApi {
         },
         {
           applyDecorators: this._decorateStory,
-          getDecorators: () => [...localDecorators, ..._globalDecorators, withSubscriptionTracking],
+          getDecorators: () => [
+            ...(allParam.decorators || []),
```

Comment:
@tmeasday not part of the change per se, but why is this called `allParam`?

Reply:
@shilman I guess it's a combination of all parameters (local global etc) in a similar way?

Reply:
This is why I meant that `addDecorator` could just be redefined to `addDecorator(d) => addParameters({ decorators: [d] })`